tmt/convert.py

```diff
@@ -689,13 +725,13 @@ def read_nitrate_case(testcase, makefile_data=None, general=False):
             data['contact'] = '{} <{}>'.format(
                 testcase.tester.name, testcase.tester.email)
         else:
-            try:
-                # Use contact from Makefile if it's there and email matches
-                if re.search(testcase.tester.email, makefile_data['contact']):
-                    data['contact'] = makefile_data['contact']
-                else:
-                    raise KeyError
-            except (KeyError, TypeError):
+            if makefile_data is None:
```

Comment:
@ukulekek I think the original code here also handles if `makefile_data` dict does not contain `contact` key?

Reply:
maybe

```suggestion
            if makefile_data is None or 'contact' not in makefile_data:
```

?